kafka/strimzi/README.md

```diff
@@ -1,30 +1,37 @@
 # Kafka Integration
 
+Seldon Core v2 relies on Kafka to implement datacentric inference Pipleines. 
+
+To install kafka in your k8s cluster, we higlight two options, ansible and helm.
+
+
 ## Install Kafka
 
-Clone https://github.com/SeldonIO/ansible-k8s-collection
+### Ansible
 
-Create a kafka operator install in kafka namespace
+We provide automation around the installation of a Kafka cluster for Seldon Core v2 to help with development and testing usecases.
 
-create playbooks/kafka_scv2.yaml
+You can follow the steps defined [here](../../ansible/README.md) to install kafka via ansible.
 
-```
-- name: Install Kafka
-  hosts: localhost
-  roles:
-    - strimzi_kafka
-  vars:
-    strimzi_kafka_create_cluster: false
-    strimzi_kafka_create_metrics: false
-```
+### Helm
+
+Use helm if you require changes and customisations to the Kafka cluster.
+
+The installation of a kafka cluster requires the strimzi operator installed in the same namespace.
```

Comment:
I am not sure it needs to be in the same namespace as usually operators tend to be clusterwide - would be good to check